chainer/functions/connection/convolution_nd.py

```diff
@@ -5,7 +5,8 @@
 import chainer
 from chainer.backends import cuda
 from chainer import configuration
-from chainer import function
+from chainer import function_node
+import chainer.functions
```

Comment:
We can probably skip this import since `chainer` is already imported.

Reply:
Fixed.

Reply:
Thanks a lot!